app/directives.js

```diff
@@ -0,0 +1,74 @@
+// Directives
+airTrafficControlApp.directive('aircraftDetails', function(){
+   return{
+      templateUrl:'directives/aircraftDetails.html',
+      scope:{
+         aircraftObject: "="
+         // this is causing two way binding, not sure

```

Comment:
If that is the case, then use parameters. If all you want this directive to know is the aircraft object and you do not want that to be associated with the scope, then simply pass the aircraft object to the directive, then the scope can contain it or not, and the directive does not care.

My point is not to try to do 'weird' things, where you are making other developers ask why are we doing this. Any hacks or things against the grain, should be added only after other alternatives are not feasible